Blog: How to think about your first A/B test (why, when, and where) (#7511)
PostHog/posthog.com: contents/docs/experiments/tutorials.mdx

- [A software engineer's guide to A/B testing](/blog/ab-testing-guide-for-engineers)
- [8 annoying A/B testing mistakes every engineer should know](/blog/ab-testing-mistakes)
- [When and how to run group-targeted A/B tests](/blog/running-group-targeted-ab-tests)

> - [How to think about your first A/B test (why, when, and where)](/product-engineers/your-first-ab-test)

Reviewer: ```suggestion
- [How to do A/B testing – a guide for beginners](/product-engineers/how-to-do-ab-testing)
```
